man/tpm2_createpolicy.1.md

```diff
@@ -46,6 +46,9 @@ These options control creating the policy authorization session:
     Start a policy session of type **TPM_SE_POLICY**. Default without this option
     is **TPM_SE_TRIAL**.
 
+  * **-C**, **--context**:_CONTEXT\_FILE_:
+    An optional file used to store the session context created by either -e or -a.
+
```

Comment:
Not sure on this. I was hoping that abrmd is getting session continuation support you would just chain everything together with -S <FILE|HANDLE>.

```
tpm2_createpolicy -S foo ....
tpm2_<command> -S foo ...
...
```
On some commands, like tpm2_load,  -C is used for object context handle save. I think I want this to be -S, so we can start moving the tools to support more and more session handling.